Library/Homebrew/os/mac/xcode.rb

```diff
@@ -78,7 +79,7 @@ def uncached_version
 
         return nil if !MacOS::Xcode.installed? && !MacOS::CLT.installed?
 
-        %W[#{prefix}/usr/bin/xcodebuild #{which("xcodebuild")}].uniq.each do |path|
+        [(prefix/"usr/bin/xcodebuild").to_s, which("xcodebuild").to_s].uniq.each do |path|
```

Comment:
This feels like one of the few places where it would be clearer to retain the `Pathname` instances (i.e. avoid the `to_s`) and to adjust the one or two code lines below.